views/js/controller/list/index.js

```diff
@@ -322,6 +332,22 @@ define([
         }
     }
 
+    function handleLoadMore() {
+        debugger;
+        const $btn  = $(this);
+        const loadMoreUrl = urlUtil.route('getListElements', 'Lists', 'taoBackOffice');
+        const uri   = getUriValue($btn.data('uri'));
+        const offset = $btn.parents('.container-content').children('ol').children('[id^=list-element]').length;
+        const limit = 20;
+        $.getJSON(
+            loadMoreUrl,
+            { uri, offset, limit },
+            response => {
+                debugger;
```

Comment:
https://github.com/oat-sa/extension-tao-backoffice/pull/116/files#r769649346

Reply:
I think, @adriloma still working on this task and will remove this asap 🙂